languagetool-language-modules/de/src/main/resources/org/languagetool/resource/de/other_prefixes.txt

```diff
@@ -0,0 +1,411 @@
+#list of prefixes comes from https://github.com/danielnaber/jwordsplitter/
+sach
+hilf
+eß
+sprach
+sozial
+freß
```

Comment:
same like "eß" -> should be removed as we would accept words like _Freßfeinde_ (old spelling)

Reply:
I think I had an old version of the german_prefix file